### DISCUSSION TITLE
Trojan detection in ESET Mobile Security

### DISCUSSION THREAD

**Discussion by FlorianKrammel** [rustdesk/rustdesk#5940] [General]
### Bug Description

ESET antivirus detects a trojan in "base.apk"

### How to Reproduce

ESET Mobile Security v8.2.15.0
Signature DB Version: 28026

Rustdesk Android: v1.1.10-1

1) Do a system scan.
2) Upload base.apk to virustotal.com

### Expected Behavior

No Virus
No Trojan
No Malware

### Operating system(s) on local side and remote side

Android 11

### RustDesk Version(s) on local side and remote side

10.1.10-1

### Screenshots

![IMG_20231007_112951](https://github.com/rustdesk/rustdesk/assets/20611911/465e008c-00e5-4808-a425-b739d9aa25e2)
![IMG_20231007_113017](https://github.com/rustdesk/rustdesk/assets/20611911/50e2dfc9-4463-48d9-8d2e-8b1d73d5404b)
![IMG_20231007_113307](https://github.com/rustdesk/rustdesk/assets/20611911/70dba1fc-4f4d-4a65-8bd5-676a01beb944)

**Comment by Darin755** [comment]
Where did you get the apk from? I would recommended installing it from F-droid or the play store

**Reply by aviv926** [reply]
same here, installed from play store
APK SHA-256:
`66196b8b9a80b322295788157becc80c08ba0badf408f0fcbdddf9386a30007c`

more apk info from [app manager](https://f-droid.org/packages/io.github.muntashirakon.AppManager/)
![image](https://github.com/rustdesk/rustdesk/assets/51673860/83d6106b-bac3-4441-a2ab-2cc035230a20)

**Comment by FlorianKrammel** [comment]
> Where did you get the apk from? I would recommended installing it from F-droid or the play store

![Screenshot_20231009-224534_Einstellungen.png](https://github.com/rustdesk/rustdesk/assets/20611911/f0567cd3-47d1-4922-ace5-0b91b7db4448)

installed from playstore

And this is the download from f-droid:
![Screenshot_20231009-224826_ESET_Mobile_Security.png](https://github.com/rustdesk/rustdesk/assets/20611911/11ee5fee-4934-468f-be38-7a1f0be1d669)

**Comment by mikekgr** [comment]
Please do somethig. The problem is getting worst now in virustotal has 11 security vendors and 2 sandboxes flagged this file as malicious
Please see:
https://www.virustotal.com/gui/file/0856e5c8c3ee94edfddb80be160ee6857ab72d972065be86783af2c907a7cd54

**Comment by Darin755** [comment]
Have you tried reporting false positives? Also it might be time to start searching the commits to try to find anything problematic

**Reply by FlorianKrammel** [reply]
I did it @eset-security on saturday.

**Comment by My1** [comment]
technically speaking it isnt wrong rustdesk can be used to control the device or send malicious code to the device, same as teamviewer, anydesk or any of these.

**Comment by FlorianKrammel** [comment]
Current state:

ESET Security has confirmed a false positive and will update their signature db.

This will solve my problem.

**Comment by paulsommer** [comment]
Maybe you should get in contact with your snake oil provider :-)

**Comment by FlorianKrammel** [comment]
Hi,

It's back, again.

"Android FakeApp.ZF" in ESET AV for Android.

**Comment by Darin755** [comment]
> Hi,
> 
> It's back, again.
> 
> "Android FakeApp.ZF" in ESET AV for Android.

I imagine this is going to be a common thing. Maybe its time to add some documentation around this issue.

**Comment by FlorianKrammel** [comment]
Hi,

after 6 month ESET AV reports malware again. 

Android/FakeApp.XC

The false positive report has been sent.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2485
